src/game.cpp

```diff
@@ -13116,53 +13161,116 @@ void game::climb_down( const tripoint &examp )
         return;
     }
 
+    bool can_use_ladder = ( height == 1 ) &&
+                          ( m.has_flag( ter_furn_flag::TFLAG_LADDER, where )
+                            || m.has_flag( ter_furn_flag::TFLAG_GOES_UP, where ) );
     bool has_grapnel = you.has_amount( itype_grapnel, 1 );
     bool web_rappel = you.has_flag( json_flag_WEB_RAPPEL );
     const int climb_cost = you.climbing_cost( where, examp );
     const float fall_mod = you.fall_damage_mod();
     add_msg_debug( debugmode::DF_IEXAMINE, "Climb cost %d", climb_cost );
     add_msg_debug( debugmode::DF_IEXAMINE, "Fall damage modifier %.2f", fall_mod );
-    const char *query_str;
-    if( !web_rappel ) {
-        query_str = n_gettext( "Looks like %d story.  Jump down?",
-                               "Looks like %d stories.  Jump down?",
-                               height );
-    } else {
-        query_str = n_gettext( "Looks like %d story.  Nothing your webs can't handle.  Descend?",
-                               "Looks like %d stories.  Nothing your webs can't handle.  Descend?", height );
-    }
 
-    if( height > 1 && !query_yn( query_str, height ) ) {
-        return;
-    } else if( height == 1 ) {
+    {
         you.set_activity_level( ACTIVE_EXERCISE );
         weary_mult = 1.0f / you.exertion_adjusted_move_multiplier( ACTIVE_EXERCISE );
 
-        if( has_grapnel ) {
-            if( !query_yn( _( "Use your grappling hook to climb down?" ) ) ) {
+        std::string hint_remaining_fall;
+
+        if( height > 1 ) {
+            // After grappling or climbing down from a 2+ high ledge, the player will free-fall.
+            // We don't assess the expected damage but this simple warning is provided.
+            hint_remaining_fall = "\n";
+            hint_remaining_fall += string_format( n_gettext(
+                    "Even if you climb down safely, you will fall <color_yellow>at least %d story</color>.",
+                    "Even if you climb down safely, you will fall <color_red>at least %d stories</color>.",
+                    height - 1 ), height - 1 );
+        }
+
+        if( can_use_ladder ) {
+            if( !query_yn( _( "Use the ladder to climb down?" ) ) ) {
```

Comment:
I would use `"Climb down the ladder?"`, because the negation to that is 'don't climb <down the ladder>', which is what happens. Negating `"Use the ladder to climb down?"` means 'Don't use the ladder to climb down' but *does* mean to climb down some other way, which is not what happens.

Or even `"There is a ladder below. Climb down the ladder?"` to make it more obvious.

Reply:
Good suggestion.  The existing implementation handled grappling hooks and webs by prompting about whether you wanted to use them, and then prompting to climb down by hand if you declined.  My phrasing was based on that, but your suggestion is better.